### PR TITLE
Document importjson format

### PR DESCRIPTION
**Pull request by andrewdotn**
This PR incorporates and elaborates on #812 and #956.

### CODE REVIEW

File: docs/dictionary-data.md [hunk 1]
```diff
@@ -4,7 +4,400 @@
```
Comment (dwhieb) [inline docs/dictionary-data.md:4]: This file uses the `sssttt` convention a few times. The explainer in linguistic-data-intro.md about source/target language is good—reuse / link to that explanation in this file too?

**Review by dwhieb — APPROVED** [review]
This will be super helpful to point linguists too, and it's written at the right level of expertise I think. Looks really good. Just a few small comments.

File: docs/dictionary-data.md [hunk 2]
```diff
@@ -4,7 +4,400 @@
...
+
+    To be clear: the FST lemma is the thing that gets plugged into a
+    paradigm layout template to generate associated wordforms. For example,
+    if the head is `nîminâniwan` with FST analysis `nîmiw+V+AI+Ind+X`, then
+    the FST lemma is `nîmiw`. If the head is `nimîw` with the FST
+    analysis `nimîw+V+AI+Ind+3Sg`, then the FST lemma is `nimîw`, which is
+    the same as the head.
+
+    Normally the FST lemma is included as part of the `analysis`, so this
+    field may only be specified on entries without an analysis.
```
Comment (dwhieb) [inline docs/dictionary-data.md:286]: Which do you mean?

* The `fstLemma` field should not be included if the `analysis` field is present.
* The `fstLemma` field may be omitted if the `analysis` field is present.

File: docs/dictionary-data.md [hunk 3]
```diff
@@ -4,7 +4,400 @@
...
+  - Run the importjson files through [`prettier`][prettier] to format them
+    nicely. If using the command line, you’ll need to add a `--parser=json`
+    argument as prettier does not automatically recognize the `.importjson`
+    file extension.
+
+  - Sort the entries by `slug`, with any `formOf` entries coming
+    immediately after the corresponding lemma entry, and related `formOf`
+    entries sorted together by `head`.
+
+    Compare strings using NFD unicode normalization so that accented and
```
[the remark this answers is not in the document]
Comment (andrewdotn) [reply docs/dictionary-data.md]: Good suggestion.

File: docs/dictionary-data.md [hunk 4]
```diff
@@ -4,7 +4,400 @@
...
+    templates have access to this data for the purpose of displaying
+    additional data to the end user.
+
+    Morphodict does not use any of this data for its core
+    language-independent functionality.
+
+    It is recommended *not* to put any unused data in here that ‘might be
+    handy later,’ but only to add new things here when required as part of
+    a coordinated effort with the frontend code to add new user-facing
+    features.
```
[the remark this answers is not in the document]
Comment (andrewdotn) [reply docs/dictionary-data.md:274]: 😄

File: docs/dictionary-data.md [hunk 5]
```diff
@@ -4,7 +4,400 @@
...
+  - Every entry must have at least one non-empty definition, and every
+    definition must have at least one valid source.
+
+  - If an `analysis` is specified, it must be one of the results returned
+    from doing an FST lookup on the `head`
+
+  - If an `analysis` is specified, the `head` must be one of the results
+    returned by running the `analysis` through the generator FST
+
+  - `fstLemma` may not be specified if there is an `analysis`
```
[the remark this answers is not in the document]
Comment (andrewdotn) [reply docs/dictionary-data.md:358]: Rewrote this section.

**Pull request merged**